README.md

```diff
@@ -1 +1,179 @@
-# To-do-List
\ No newline at end of file
+
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <h2><b> To-do-List App</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 To-do-List <a name="about-project"></a>
+
+**To do list** is a repository.In this project,In this project, you will build a simple HTML list of To Do tasks. This simple project has been built using webpack and served by a webpack dev server.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">css</a></li>
+    <li><a href="https://www.w3schools.com/js/">Javascript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Form backend service</summary>
+  <ul>
+    <li><a href="https://formspree.io/html/">formspree</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Demo of Linters**
+- **Demo of To-Do-List**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://github.com/Eccb7/To-do-List/)
```

Comment:
- [ ] Your live demo link is broken. This link should lead to a deployed version of your project but it does not. Please host the project and update the link or delete the entire live demo section if you are required to host the project  💭